test/files/neg/warn-unused-params.check

```diff
@@ -13,6 +13,12 @@ case class CaseyAtTheBat(k: Int)(s: String)        // warn
 warn-unused-params.scala:62: warning: parameter value readResolve in method f is never used
   def f(readResolve: Int) = 42           // warn
         ^
+warn-unused-params.scala:76: warning: parameter value i in value $anonfun is never used
```

Comment:
... "in this anonymous function is never used" if `p.owner.isAnonymousFunction`?

Reply:
I'll wait for final tweaking. The goal for the next push is to track usage during `for` desugaring or caramelization.

Reply:
Done on linked PR.